### PR TITLE
created project

### PR DESCRIPTION
**Pull request by alex625144**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.7.10-SNAPSHOT</version>
```
Comment (FedosovMax) [inline pom.xml:8]: Why not use 3+ version? We aren't going to have a lot of other integrations so it shouldn't be a lot of problems.

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.neprozorro</groupId>
+    <artifactId>demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>demo</name>
+    <description>Demo project for Spring Boot</description>
+    <properties>
+        <java.version>1.8</java.version>
```
Comment (FedosovMax) [inline pom.xml:17]: we need 17 version, 8 is too old for this shit

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>org.postgresql</groupId>
+            <artifactId>postgresql</artifactId>
+            <scope>runtime</scope>
```
Comment (FedosovMax) [inline pom.xml:32]: why is it runtime dependency?

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <optional>true</optional>
```
Comment (FedosovMax) [inline pom.xml:37]: Why is it optional?

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+        </repository>
+        <repository>
+            <id>spring-snapshots</id>
+            <name>Spring Snapshots</name>
+            <url>https://repo.spring.io/snapshot</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+    </repositories>
```
Comment (FedosovMax) [inline pom.xml:79]: What is it and why do we need it?

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+        </pluginRepository>
+        <pluginRepository>
+            <id>spring-snapshots</id>
+            <name>Spring Snapshots</name>
+            <url>https://repo.spring.io/snapshot</url>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </pluginRepository>
+    </pluginRepositories>
```
Comment (FedosovMax) [inline pom.xml:97]: Why do we need this?

File: src/main/resources/application.yml [hunk 7]
```diff
@@ -0,0 +1,29 @@
+server:
+  port:
```
Comment (FedosovMax) [inline src/main/resources/application.yml:2]: what port? What will happen if we have this field empty? Why do we need it at all if it's empty?

File: src/main/resources/application.yml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+logging:
+  level:
+    com.knighttodo.knighttodo: INFO
```
Comment (FedosovMax) [inline src/main/resources/application.yml:6]: you should change the package

File: src/main/resources/application.yml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+
+spring:
+  output:
+    ansi:
+      enabled: NEVER
```
Comment (FedosovMax) [inline src/main/resources/application.yml:11]: What does it mean?

File: src/main/resources/application.yml [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+  datasource:
+    url: jdbc:postgresql://localhost:5432/postgres?currentSchema=neprozorro
+    username: postgres
+    password: 123
+    driver-class-name: org.postgresql.Driver
+  jpa:
+    hibernate:
+      ddl-auto: create
```
Comment (FedosovMax) [inline src/main/resources/application.yml:19]: it's better to use validate by default, other people will change it to create for creating DB

File: src/main/resources/application.yml [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+    show-sql: false
+    properties:
+      hibernate:
+        dialect: org.hibernate.dialect.PostgreSQLDialect
+        format_sql: true
+        order_by:
+          default_null_ordering: last
+
+baseUrl:
+  experience: "http://localhost:6070"
```
Comment (FedosovMax) [inline src/main/resources/application.yml:29]: you don't need it

File: pom.xml (hunk 6 above)
Comment (alex625144) [reply pom.xml:77]: It was generated spring initializr. I removed it.

File: pom.xml (hunk 1 above)
Comment (alex625144) [reply pom.xml]: Changed.

File: pom.xml (hunk 2 above)
Comment (alex625144) [reply pom.xml]: Done.

File: pom.xml (hunk 3 above)
Comment (alex625144) [reply pom.xml]: It was generated. I removed.

File: pom.xml (hunk 5 above)
Comment (alex625144) [reply pom.xml]: I removed.

File: src/main/resources/application.yml (hunk 7 above)
Comment (alex625144) [reply src/main/resources/application.yml]: If empty it means 8080.

File: src/main/resources/application.yml (hunk 8 above)
Comment (alex625144) [reply src/main/resources/application.yml]: Done.

File: src/main/resources/application.yml (hunk 9 above)
Comment (alex625144) [reply src/main/resources/application.yml]: It ansi (colour output) in spring.

File: src/main/resources/application.yml (hunk 10 above)
Comment (alex625144) [reply src/main/resources/application.yml]: Ok

File: src/main/resources/application.yml (hunk 11 above)
Comment (alex625144) [reply src/main/resources/application.yml]: Ok

File: pom.xml [hunk 12]
```diff
@@ -0,0 +1,60 @@
...
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>3.0.3</version>
+        <relativePath/> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.neprozorro</groupId>
+    <artifactId>demo</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>demo</name>
+    <description>Demo project for Spring Boot</description>
```
Comment (FedosovMax) [inline pom.xml:15]: Consider using a better description and artifact name, change Demo to something more descriptive please. Everything should make sense or be removed.

**Review by FedosovMax — APPROVED** [review]

**Pull request merged**